components/rmrk/ExploreLayout/ExploreLayout.vue

```diff
@@ -4,74 +4,80 @@
       {{ $t('general.searchResultsText') }}
       <span class="text__stroked is-size-3">{{ $route.query.search }}</span>
     </div>
-    <o-tabs
-      v-model="selectedTab"
+    <NeoTab
+      :v-model="selectedTab"
       data-cy="tabs"
       class="explore-tabs"
-      content-class="explore-tabs-content">
-      <o-tab-item :label="$t('collections')" value="collectibles">
-        <template v-if="selectedTab === 'collectibles'">
+      content-class="explore-tabs-content"
+      @update:modelValue="updateTab($event)">
+      <template #items>
+        <NeoTabItem :label="$t('collections')" value="collectibles">
           <CollectionList />
-        </template>
-      </o-tab-item>
+        </NeoTabItem>
 
-      <o-tab-item :label="$t('gallery')" value="items" class="p-5">
-        <Gallery v-if="selectedTab === 'items'" />
-      </o-tab-item>
-    </o-tabs>
+        <NeoTabItem :label="$t('gallery')" value="items" class="p-5">
+          <Gallery />
+        </NeoTabItem>
+      </template>
+    </NeoTab>
   </div>
 </template>
 
 <script setup lang="ts">
-import { OTabItem, OTabs } from '@oruga-ui/oruga'
-
+import { NeoTab, NeoTabItem } from '@kodadot1/brick'
 const { $route, $router } = useNuxtApp()
+let selectedTab
 
-const selectedTab = computed({
-  // getter
-  get() {
-    return $route.name?.split('-')[2] as string
-  },
-  // setter
-  set(val) {
-    $route.query.page = ''
-    let queryOptions: {
-      page: string
-      search?: string | (string | null)[]
-    } = {
-      page: '1',
-    }
-    if ($route.query.search) {
-      queryOptions.search = $route.query.search
-    }
+// const selectedTab = computed({
+//   // getter
+//   get() {
+//     console.log('helllo')
+//     return $route.name?.split('-')[2] as string
+//   },
+//   // setter
+//   set(val) {
+//     console.log('helllo2', $route.query.page)
 
-    $router.replace({
-      path: val,
-      query: queryOptions,
-    })
-  },
-})
-</script>
-<style lang="scss">
-@import '@/styles/abstracts/variables';
+//     $route.query.page = ''
+//     let queryOptions: {
+//       page: string
+//       search?: string | (string | null)[]
+//     } = {
+//       page: '1',
+//     }
+//     if ($route.query.search) {
+//       queryOptions.search = $route.query.search
+//     }
 
-.explore-tabs {
-  border: none;
-  box-shadow: none;
+//     $router.replace({
+//       path: val,
+//       query: queryOptions,
+//     })
+//   },
+// })
 
-  .o-tabs__nav {
-    margin-bottom: 35px;
-    max-width: 450px;
-    box-shadow: $primary-shadow;
+const updateTab = (val) => {
```

Comment:
wanted to do this through getter and setter, the above commented part, but wasnt able to make it work, if someone knows how to, please let me know